src/styles/public/sites/cleanbc/_cleanbc--navigation-override.scss

```diff
@@ -43,7 +47,6 @@ body.cleanbc {
 					// border: 1px dashed var(--wp--preset--color--senary);
```

Comment:
Is this commented out code needed?

Reply:
Yes, for the moment I am keeping that in. It will be removed at a later date.